GooglePlayInstant/Editor/PlayInstantAssetBundleBrowserClient.cs

```diff
@@ -0,0 +1,151 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.IO;
+using System.Linq;
+using System.Text.RegularExpressions;
+using UnityEditor;
+using UnityEngine;
+
+namespace GooglePlayInstant.Editor
+{
+    /// <summary>
+    /// A static class responsible for interactions with the Asset Bundle Browser
+    /// <see cref="https://github.com/Unity-Technologies/AssetBundles-Browser"/>, such as detecting whether the browser
+    /// is present or getting the current version of the Asset Bundle Browser.
+    /// </summary>
+    public static class AssetBundleBrowserClient
+    {
+        private const string AssetBundleBrowserPackageName = "com.unity.assetbundlebrowser";
+        private const string AssetBundleBrowserMenuItem = "Window/AssetBundle Browser";
+        private static bool? _assetBundleBrowserIsPresent;
+        private static string _assetBundleBrowserVersion;
+
+        /// <param name="useCurrentValueIfPresent">A boolean value corresponding to whether the caller wants to
```

Comment:
Wrap the whole thing in a summary block. I'm not sure the param makes sense without that.